src/python/pants/backend/python/target_types.py

```diff
@@ -404,25 +415,39 @@ class PexExecutionMode(Enum):
     VENV = "venv"
 
 
-# TODO(John Sirois): Deprecate UNZIP mode: https://github.com/pantsbuild/pants/issues/12803
 class PexExecutionModeField(StringField):
     alias = "execution_mode"
     valid_choices = PexExecutionMode
     expected_type = str
     default = PexExecutionMode.ZIPAPP.value
     help = (
-        "The mode the generated PEX file will run in.\n\nThe traditional PEX file runs in "
-        f"{PexExecutionMode.ZIPAPP.value!r} mode (See: https://www.python.org/dev/peps/pep-0441/). "
-        f"In general, faster cold start times can be attained using the "
-        f"{PexExecutionMode.UNZIP.value!r} mode which also has the benefit of allowing standard "
-        "use of `__file__` and filesystem APIs to access code and resources in the PEX.\n\nThe "
-        f"fastest execution mode in the steady state is {PexExecutionMode.VENV.value!r}, which "
-        "generates a virtual environment from the PEX file on first run, but then achieves near "
-        "native virtual environment start times. This mode also benefits from a traditional "
-        "virtual environment `sys.path`, giving maximum compatibility with stdlib and third party "
-        "APIs."
+        "The mode the generated PEX file will run in.\n\nThe traditional PEX file runs in a "
+        f"modified {PexExecutionMode.ZIPAPP.value!r} mode (See: "
+        "https://www.python.org/dev/peps/pep-0441/) where zipped internal code and dependencies "
+        "are first unpacked to disk. This mode achieves the fasted cold start times and may, for "
```

Comment:
```suggestion
        "are first unpacked to disk. This mode achieves the fastest cold start times and may, for "
```

Reply:
The degree to which I hate language is unmeasurable.

Reply:
Thanks for the --not-zip-safe catch. I failed to search for that. Fixed.